extensions/MongoDbAtlas/MongoDbAtlas/MongoDbAtlasStorage.cs

```diff
@@ -105,7 +107,8 @@ public Task DeleteDocumentDirectoryAsync(string index, string documentId, Cancel
                 {
                     { "index", index },
                     { "documentId", documentId },
-                    { "fileName", fileName }
+                    { "fileName", fileName },
+                    { "contentType", (new Pipeline.MimeTypesDetection()).GetFileType(fileName)}
```

Comment:
this hardcodes the MimeTypesDetection class, which blocks from injecting a different implementation. Could you use DI, injecting the dependency in the ctor? You can reuse the code from `AzureBlobsStorage`

<img width="371" alt="image" src="https://github.com/microsoft/kernel-memory/assets/371009/7db9f081-b183-4ab2-b65b-700703f093ef">

Reply:
Got it. Thanks for the reference.